egs/wsj/s5/steps/nnet3/make_cnn_tdnn_configs.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import re, os, argparse, sys, math, warnings
+import numpy as np
```

Comment:
Agreed.  I only allow numpy in the reporting stuff because the reporting is
an optional feature.   I'd rather avoid any dependency on it in critical
parts of the recipes.
Dan


On Mon, Feb 22, 2016 at 6:02 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/make_cnn_tdnn_configs.py
> <https://github.com/kaldi-asr/kaldi/pull/517#discussion_r53709162>:
>
> > @@ -0,0 +1,397 @@
> > +#!/usr/bin/env python
> > +
> > +# we're using python 3.x style print but want it to work in python 2.x,
> > +from __future__ import print_function
> > +import re, os, argparse, sys, math, warnings
> > +import numpy as np
>
> BTW I don't think numpy is critical here. We usually avoid importing numpy
> unless it is absolutely necessary. Could you replace your 2D-numpy arrays
> with list of lists ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/517/files#r53709162>.
>